content/docs/0.13.x/operate/user_management/openid_authentication/index.md

```diff
@@ -4,10 +4,10 @@ description: How to setup user authentication using OIDC
 weight: 10
 ---
 
-To set up user authenticating using OIDC, customization of your **Keptn Bridge** is needed. 
+Please follow the following links for instructions how to setup user authentication using OIDC:
 
-* Please follow the instructions provided [here](../../../reference/bridge/oauth/#enable-disable-authentication).
+* [Keptn Bridge](../../../reference/bridge/oauth/#enable-disable-authentication)
+* [Keptn CLI](../../../reference/cli/commands/keptn_auth)
```

Comment:
This link does not work

Reply:
in the netlify preview they seem to work

Reply:
it just gives me github 404 site when I click on it here in github, but when adding ".md" to the end, it works

Reply:
try here https://github.com/keptn/keptn.github.io/blob/e5ccbe837b4f3fe2e06cf7f1f813944c5fe546c7/content/docs/0.13.x/operate/user_management/openid_authentication/index.md

Reply:
you need to try here: https://deploy-preview-1072--keptn.netlify.app/docs/0.13.x/operate/user_management/openid_authentication/

Reply:
These Markdown files are compiled into a website using [Hugo](https://gohugo.io/). That's why your link doesn't work @odubajDT  :)